feature-libs/product-configurator/rulebased/components/attribute/header/configurator-attribute-header.component.ts

```diff
@@ -136,4 +136,15 @@ export class ConfiguratorAttributeHeaderComponent
     const images = this.attribute.images;
     return images && this.hasImage ? images[0] : undefined;
   }
+
+  /**
+   * Creates unique key for config attribute on the UI.
+   *
+   * @param {string} prefix - Prefix for key depending on usage (e.g. uiType, label)
+   * @param {string} attributeId - Attribute ID
+   * @return {string} - Generated attribute UI key
+   */
+  createAttributeUiKey(prefix: string, attributeId: string): string {
```

Comment:
is it better to do this?
```
  get attributeUiKey(): string  {
    return this.uiKeyGeneratorService.createAttributeUiKey('label', this.attribute.name);
  }
```
use `attributeUiKey` in html directly.

Reply:
Done!